app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -29,6 +29,22 @@ def textual_group_smart_management
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
+  def textual_group_volumes
+    labels = [_("Name"), _("Property"), _("Value")]
+    h = {:labels => labels}
+    h[:values] = []
+    @record.container_volumes.each do |volume|
```

Comment:
This should produce a more human-friendly output, let's do:

    key_dictionary = [
        [:empty_dir_medium_type, _('Empty Directory')],
        [:gce_pd_name, _('GCE Persistent Disk')],
        ...
    ]

    h = {:labels => [_("Name"), _("Property"), _("Value")], :values => []}

    @record.container_volumes.each do |volume|
        volume_values = key_dictionary.collect do |key, name|
            [nil, name, volume[key]] if volume[key].present?
        end.compact
        # Set the volume name only  for the first item in the list
        volume_values[0][0] = volume.name if volume_values.length > 0
        h[:values] += volume_values
    end